lib/TimeZoneUtil.php

```diff
@@ -136,6 +136,10 @@ public static function getTimeZone($tzid, Component $vcalendar = null, $failIfUn
         // this method will return just GMT+01:00. This is wrong, because it
         // doesn't take DST into account.
         if ('(' !== $tzid[0]) {
+            // If the timezone is prefixed with a slash and is not defined in the vcalendar => we remove the slash.
+            if ('/' === $tzid[0] && !$vcalendar) {
```

Comment:
Why do you add check for !$vcalendar?

Reply:
Otherwise `TZID=/freeassociation.sourceforge.net/Tzfile/SystemV/EST5EDT` will fail